tests/test_concurrency/test_mainloopscheduler/test_tkinterscheduler.py

```diff
@@ -5,9 +5,9 @@
 from rx.concurrency.mainloopscheduler import TkinterScheduler
 
 tkinter = pytest.importorskip("tkinter")
-
+import tkinter
```

Comment:
Ugh, sorry, this import should not have been added. I just pushed an extra commit to remove it.